src/napari_epitools/_reader.py

```diff
@@ -81,12 +81,8 @@ def reader_function(path):
     for layer in image_layers:
         layer_data, layer_kwargs, layer_type = layer
         layer_kwargs["metadata"] = {
-            "scale": np.asarray(
-                layer_kwargs.pop("scale")
-            ),  # don't scale the image in the viewer
-            "spacing": np.asarray(
-                image_stack.image.spacing
-            ),  # we need this for regionprops
+            "yx_spacing": np.asarray(
+                image_stack.image.spacing[1:]
+            ),  # ZYX[1:], we need this spacing for regionprops
```

Comment:
More readable
```suggestion
            # ZYX[1:], we need this spacing for regionprops
            "yx_spacing": np.asarray(image_stack.image.spacing[1:]),
```

Reply:
I've moved the commend to the top of the block and expanded it a bit